src/main/flight/mixer_init.c

```diff
@@ -369,6 +369,7 @@ void mixerInitProfile(void)
 
     mixerRuntime.ezLandingThreshold = 2.0f * currentPidProfile->ez_landing_threshold / 100.0f;
     mixerRuntime.ezLandingLimit = currentPidProfile->ez_landing_limit / 100.0f;
+    mixerRuntime.ezLandingSpeed = currentPidProfile->ez_landing_speed > 0 ? currentPidProfile->ez_landing_speed / 10.0f : 0.0f;
```

Comment:
The scaling here is wrong and will effectively turn off EzLanding at half the speed of that which it is supposed to, fixing now.